src/foraging_gui/TransferToNWB.py

```diff
@@ -129,9 +131,9 @@ class obj:
     # need to get more subject information through subject_id
     nwbfile.subject = Subject(
         subject_id=obj.ID,
-        description='Animal name:'+obj.ID+'  Weight after(g):'+obj.WeightAfter,
+        description='Animal name:'+obj.ID,
         species="Mus musculus",
-        weight=obj.WeightBefore,
+        weight=_get_field(obj, 'WeightAfter'),
```

Comment:
"WeightBefore" existed in the old data type, and is gone now.

Reply:
So we decided to use weightafter as subject.weight for all data, leaving the field of old data np.nan, right? Because we should be consistent about the naming.

Reply:
I remember that for the old data we also have the `WeightAfter` (I checked session 668551_2023-06-16.json).

Reply:
I set the default as np.nan